cypress.json

```diff
@@ -0,0 +1,7 @@
+{
+  "baseUrl": "http://localhost:3000",
+  "projectId": "x99ctf",
```

Comment:
See https://dashboard.cypress.io/projects/x99ctf - plan to add this link to our docs once this is merged